src/App/Routes/Modals/RegisterModal/View.tsx

```diff
@@ -39,7 +39,7 @@ const RegisterModal: FC<Props> = ({ account, displayModal, githubHandle, onClose
     return (
       <>
         <p className="mt-9 mb-11 font-alfreda font-bold text-5xl">If you wish to contribute, connect your Wallet</p>
-        <ConnectButton>Connect Wallet</ConnectButton>
+        <MultiWalletConnection>Connect</MultiWalletConnection>
```

Comment:
The component does not have children anymore, it imposes its own render so you might just put `<MultiWalletConnection />`